blender_addons/io_md5_exporter/io_export_md5.py

```diff
@@ -822,9 +841,9 @@ def treat_bone(b, parent = None):
       if len(ANIMATIONS) > 0:
         anim = ANIMATIONS.popitem()[1] #ANIMATIONS.values()[0]
         createDirectoryForFileIfMissing(animationFilePath)
-        animationFilePath = os.path.join(os.path.dirname(animationFilePath),getSuggestedModelName()+a.name+".md5anim")

```

Comment:
`bearMoveRight.md5anim` for sure. In this case you know you can safely uppercase the first letter of the action name ("moveRight" to "MoveRight") since you're appending it to something else :-)